BundID/BundIDApp.swift

```diff
@@ -51,6 +51,7 @@ struct BundIDApp: App {
                 idInteractionManager: idInteractionManager,
                 storageManager: storageManager,
                 analytics: MatomoAnalyticsClient(siteId: "2", baseURL: matomoUrl),
+                urlOpener: { UIApplication.shared.open($0) },
```

Comment:
(but not a must, just nice to have)

Reply:
That does not work, was also my first approach. The [method](https://developer.apple.com/documentation/uikit/uiapplication/1648685-open) has more parameters and cannot be assigned because of that (even though they have default values)

Reply:
Good catch, sorry!